### PR TITLE
[PEPPER-804] Alternate approach.

### PR DESCRIPTION
**Pull request by otchet-broad**
@aweng98 and I have been discussing the implementation to fix [PEPPER-804](https://broadworkbench.atlassian.net/browse/PEPPER-804).  Here's an alternate approach that I was able to verify on my local machine.

## Checklist

- [X] I have labeled the type of changes involved using the `C-*` labels.
- [ ] I have assessed potential risks and labeled using the `R-*` labels.
- [ ] I have considered error handling and alerts, and added `L-*` labels as needed.
- [ ] I have considered security and privacy, and added `I-*` labels as needed
- [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [X] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [X] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [X] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

[PEPPER-804]: https://broadworkbench.atlassian.net/browse/PEPPER-804?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by aweng98 — APPROVED** [review]
GREAT!

**Review by rfricke-asymmetrik — APPROVED** [review]
LGTM

File: pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/util/TestDataSetupUtil.java [hunk 1]
```diff
@@ -613,16 +612,17 @@ public static UserProfile createTestingProfile(Handle handle, long userId, boole
...
+                        random ? rand.ints(1, 12).findFirst().orElseThrow()
                                 : TestConstants.TEST_USER_PROFILE_BIRTH_MONTH,
-                        random ? rand.ints(1, 28).findFirst().getAsInt()
+                        random ? rand.ints(1, 28).findFirst().orElseThrow()
                                 : TestConstants.TEST_USER_PROFILE_BIRTH_DAY))
                 .preferredLangId(LanguageStore
                         .get(TestConstants.TEST_USER_PROFILE_PREFERRED_LANGUAGE)
                         .getId())
                 .preferredLangCode(null)
+                .timeZone(ZoneId.of("America/New_York"))
```
Comment (otchet-broad) [inline pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/util/TestDataSetupUtil.java:625]: Set the timezone for the test user.

File: pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/GetActivityInstanceRouteStandaloneTest.java [hunk 2]
```diff
@@ -1115,12 +1116,12 @@ public void testStudyAdmin_canRetrieveInstances() {
...
-
         String expectedPrompt = "What is " + profile.getFirstName() + "'s favorite color?";
         resp.then().assertThat().body("sections[1].blocks[2].question.prompt", equalTo(expectedPrompt));
-
-        String expectedBody = String.format("<p>%s<br/>%s<br/>%s</p>", profile.getFirstName(), profile.getLastName(),
-                DateTimeFormatter.ofPattern("MM-dd-uuuu").format(LocalDate.now()));
+        String expectedBody = String.format("<p>%s<br/>%s<br/>%s</p>",
+                profile.getFirstName(),
+                profile.getLastName(),
+                DateTimeFormatter.ofPattern("MM-dd-uuuu").format(LocalDateTime.ofInstant(Instant.now(), profile.getTimeZone())));
```
Comment (otchet-broad) [inline pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/GetActivityInstanceRouteStandaloneTest.java:1124]: Use the timezone from the profile.

**Pull request merged**